service.go

```diff
@@ -343,6 +362,19 @@ func (self *Service) SetDescription(description string) error {
 	return nil
 }
 
+func (self *Service) SetDelayedAutoStart(delayedAutostart bool) error {
```

Comment:
The discrepancy between `GetDelayedAutoStartInfo` vs. `SetDelayedAutoStart` is less than ideal.  Let's pick either a structure or a Boolean and stick with it for both functions.